packages/core/src/audio/AudioSource.ts

```diff
@@ -0,0 +1,245 @@
+import { deepClone, ignoreClone } from "../clone/CloneManager";
+import { Component } from "../Component";
+import { Entity } from "../Entity";
+import { AudioClip } from "./AudioClip";
+import { AudioManager } from "./AudioManager";
+
+/**
+ * Audio Source Component.
+ */
+export class AudioSource extends Component {
```

Comment:
AudioSource looks more like an asset rather than a component

Reply:
`AudioSource` and `AudioListener` is OK! Source here is the source of the sound